js/gravityforms-mfi-multifile.js

```diff
@@ -0,0 +1,290 @@
+/*
+Copyright 2009-2016 Rocketgenius, Inc.
+
+This program is free software; you can redistribute it and/or modify
+it under the terms of the GNU General Public License as published by
+the Free Software Foundation; either version 3 of the License, or
+(at your option) any later version.
+
+This program is distributed in the hope that it will be useful,
+but WITHOUT ANY WARRANTY; without even the implied warranty of
+MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+GNU General Public License for more details.
+
+You should have received a copy of the GNU General Public License
+along with this program.  If not, see http://www.gnu.org/licenses.
+*/
+
+//----------------------------------------
+//------ MULTIFILE UPLOAD FUNCTIONS ------
+//----------------------------------------
+
+(function (gfMultiFileUploader, $) {
+    gfMultiFileUploader.uploaders = {};
+    var strings = typeof gform_gravityforms != 'undefined' ? gform_gravityforms.strings : {};
+    var imagesUrl = typeof gform_gravityforms != 'undefined' ? gform_gravityforms.vars.images_url : "";
+
+    $(document).unbind('gform_post_render');
```

Comment:
Here we unbind the original gravity forms setup function hooked to the document's 'gform_post_render' function.
The fact that this script depends on 'gform_gravityforms' script (cf. https://github.com/tyxla/Gravity-Forms-Multiple-Form-Instances/pull/24/files#diff-7d573d66d37a99e5734d905449d49972R130), ensures that the unbind will be properly executed after the original binding has been done.